.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,36 @@
+name: Renovate
+on:
+  # schedule:
+  #   # The "*" (#42, asterisk) character has special semantics in YAML, so this
+  #   # string has to be quoted.
+  #   - cron: '0/15 * * * *'
+  workflow_dispatch:
+    inputs:
+      name:
+        type: string
+        default: ""
+        description: Dummy value
+
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get token
+        uses: actions/create-github-app-token@f4c6bf6752984b3a29fcc135a5e70eb792c40c6b # v1.8.0
+        id: get_token
+        with:
+          app-id: ${{ vars.RENOVATE_APP_ID }}
+          private-key: ${{ secrets.RENOVATE_APP_PRIVATE_KEY }}
+
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@2d90417499f45ff78a09586f7b9874b19817dba3 # v40.1.0
+        with:
+          configurationFile: .github/configs/renovate-config.js
+          renovate-version: 37.192.1-slim
```

Comment:
ironigcally, you'll need renovate to keep this up to date :)

Reply:
Implemented 😎 :
![grafik](https://github.com/argoproj/argo-helm/assets/7290987/3e47c994-4a30-466c-adac-5fd6f2e9fac8)